mcache_test.go

```diff
@@ -18,7 +18,7 @@ func TestMessageCache(t *testing.T) {
 	}
 
 	for i := 0; i < 10; i++ {
-		mcache.Put(msgs[i])
+		mcache.Put(&Message{Message: msgs[i]})
```

Comment:
To avoid changing `makeTestMessage`, but can do if considered good.

Reply:
maybe it will save some characters? Up to you, doesn't really matter.